examples/example-plugin-custom-tooltip.html

```diff
@@ -128,6 +128,7 @@ <h2>View Source:</h2>
         id: "title", name: "Title", field: "title", width: 70, cssClass: "cell-title",
         editor: Slick.Editors.Text, validator: requiredFieldValidator,
         customTooltip: {
+          position: 'right', // defaults to "auto", available options are: "auto", "top", "left", "bottom", "right"
```

Comment:
change to `align-left` and `align-right` since it's not really a cell left/right position